### DISCUSSION TITLE
Icon request: icon-dancer

### DISCUSSION THREAD

**Discussion by keylabarrros** [FortAwesome/Font-Awesome#21174] [Icon Requests]
Hi,
My sugestion for you is make icons for entertainament:
icon-single dance
icon-couple dance
icon-theater (could be just curtains)

**Comment by tagliala** [comment]
https://github.com/FortAwesome/Font-Awesome/blob/master/CONTRIBUTING.md#requesting-new-icons

> Please search to see if your icon request already exists. If a request is found, please +1 that one.
> Please make requests for single icons, unless you are requesting a couple of strictly related icons (e.g., thumbs-up/thumbs-down).
> Title your new issue Icon request: icon-name (e.g., Icon request: icon-car).

Theater: #1270 #153
Dancing couple: #1082

Leaving this open as `Icon request: icon-dancer` but please provide an example

**Comment by Crissov** [comment]
- 💃 [U+1F483](http://emojipedia.org/dancer/) Unicode emoji #222 
- 🕺 [U+1F57A](http://emojipedia.org/man-dancing/)

**Comment by vladimirsolovey** [comment]
Perhaps, high heel shoe?

https://www.google.com/search?q=high+heels+icon&espv=2&biw=1536&bih=760&tbm=isch&imgil=r4gvAR0H0blNNM%253A%253BYLpVNV6VWYIzmM%253Bhttps%25253A%25252F%25252Fthenounproject.com%25252Fterm%25252Fhigh-heels%25252F168014%25252F&source=iu&pf=m&fir=r4gvAR0H0blNNM%253A%252CYLpVNV6VWYIzmM%252C_&usg=__oxmveFRmjRm8tVxNv-3eaRTtihY%3D&ved=0ahUKEwj5v-7_xp_SAhUjl1QKHT5hDboQyjcIOQ&ei=-1arWLnWEKOu0gK-wrXQCw#imgrc=r4gvAR0H0blNNM:

I could use any kind of dance icon for citywide directory (e.g. dance-studios)

**Comment by dewolft** [comment]
+1

**Comment by kristin-lynch** [comment]
These are great, but may I request a couple of icons that are a little more generic? I needed one for a catholic high school holiday dance concert and these wouldn't have been appropriate. ;)

**Comment by JacobCWard** [comment]
:+1:

**Comment by mbomb007** [comment]
+1

**Comment by EvanCarroll** [comment]
The other requests are too targeted. A couple dancing is useful, but this is a better proposal.

"icon-single dance" sounds like it would cover strip dancing, or pole dancing at least somewhat. This would be useful for the i3 editor for OSM where Strip Clubs are included. In addition to bringing a better icon than a cocktail glass for a regular nightclub.

**Comment by krkxone** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
